src/lib/forms-validation/index.ts

```diff
@@ -142,6 +146,106 @@ function validateFormElementReferences(formElements: FormTypes.FormElement[]) {
   }
 }
 
+const validAddressReferenceTypes: FormTypes.FormElement['type'][] = [
+  'pointAddress',
+  'geoscapeAddress',
+]
+const validNumericReferenceTypes: FormTypes.FormElement['type'][] = [
+  'number',
+  'calculation',
+]
+
+function validateConditionallyShowPredicateReferencedElement({
```

Comment:
Not checking for the `FORM` case?

Reply:
It's just an Id, I think the API does, but there is no cross definition reference there for the sdk to use

Reply:
Forgot about that lil tidbit on the end there 👍